renovate.json

```diff
@@ -1,47 +1,35 @@
 {
-  "packageRules": [{
-    "packageNames": [
-      "com.coveo:fmt-maven-plugin",
-      "com.google.googlejavaformat:google-java-format"
-    ],
-    "enabled": false
-  },
-    {
-      "major": {
-        "enabled": false
-      }
-    },
+  "packageRules": [
     {
       "matchPackageNames": ["org.springframework.boot:spring-boot-dependencies"],
-      "allowedVersions": "<3.0.0"
+      "allowedVersions": "<4.0.0"
```

Comment:
I feel this change does not match the compatibility matrix. 
![image](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/assets/28604/e74fdee2-4428-4c03-89a9-34cd81398d16)

![image](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/assets/28604/0f51a1e6-516e-4cce-a653-43cb3b2144e9)

Reply:
oh yeah sorry, my bad. I guess I configured them correctly before. this new commit was wrong.